DKT/prepare_sequences_test.py

```diff
@@ -0,0 +1,87 @@
+import pandas as pd
+
+from prepare_sequences import prepare_df
+from pandas.testing import assert_frame_equal
+
+df = pd.DataFrame(
```

Comment:
it's a bit weird to put the df outside the test function if there is only one test function

Reply:
I plan to add more after